runtime.osx-arm64.runtime.native.System.IO.Ports missing

### Description

There isn't a nuget package for System.IO.Ports native runtime for osx-arm64.

### Configuration

.net 6-RC1
OSX - 11.5.2
Macbook Air - Apple Silicon

### Regression?

Apple Silicon support is a new feature for .net 6

### Other information

I was able to build the native libs on my M1 machine and they built correctly. However, I can't for the life of me figure out how to build the nupkg for native libs. I've read all the various documentation, tried with dotnet pack and  nuget on every .proj I could find and VS-mac to no avail.

This document has a great section on building the native components:
https://github.com/dotnet/runtime/tree/v6.0.0-rc.1.21451.13/docs/workflow/building/libraries

But, it says nothing about how to package them. I assume it's easy and I'm just failing at finding it.

@ViktorHofer negative. It fails with:

/Users/sean/Projects/runtime/src/libraries/System.IO.Ports/pkg/runtime.osx-arm64.runtime.native.System.IO.Ports.proj : error NU1101: Unable to find package runtime.osx-arm64.runtime.native.System.IO.Ports. No packages exist with this id in source(s): dotnet-eng, dotnet-public, dotnet-tools, dotnet6, dotnet6-transport, richnav [/Users/sean/Projects/runtime/Build.proj]

It's a chicken/egg problem of some sort.

--- a/src/libraries/System.IO.Ports/pkg/runtime.osx-arm64.runtime.native.System.IO.Ports.proj
+++ b/src/libraries/System.IO.Ports/pkg/runtime.osx-arm64.runtime.native.System.IO.Ports.proj
@@ -1,0 +1,7 @@
+<Project>
+  <Import Project="runtime.native.System.IO.Ports.props" />
+  <PropertyGroup>
+    <!-- TODO: Remove when the package shipped. -->
+    <DisablePackageBaselineValidation>true</DisablePackageBaselineValidation>
+  </PropertyGroup>
+</Project>

